### ISSUE TITLE
Domain & Sub paths

### ISSUE THREAD

**Issue by Darren-CheckPlanningData** [digital-land/submit#285]
### Acceptance criteria (outcomes expected that make this work complete)

- [ ] Redirect mechanism must support both http and https traffic
- [ ] Redirect mechanism must support report usage metrics to Cloudfront (to aid with decommissioning in future)

### Tasks (list of todo's for this work)

- [x]  Agree new domain name
- [ ]  Rename Domain
- [ ]  Work out sub paths for the pages,  to avoid doing redirects later down the line

### Supporting artefacts (what previous work / current work / discussions / etc, that will help complete this work)

Domain has been changed previously by Infrastructure team:  https://trello.com/c/JxJGSel3

Direction from Owen:
With. both. Chris C and Ernest off w/c 19th Aug  it’s better to leave this work until next week (w/c 26th Aug)   as DNS changes can often have unintended consequences.
They will be able to action it fairly quickly on Monday.
My understanding is that this isn’t a super urgent change?
What we’ll need from you guys is:
a confirmation on what the new subdomain is (I think it’s [submit.planning.data.gov.uk](http://submit.planning.data.gov.uk/))
and where [check.planning.gov.uk](http://check.planning.gov.uk/) needs redirecting to I believe it’s [submit.planning.data.gov.uk/check](http://submit.planning.data.gov.uk/check)

**Comment by GeorgeGoodall** [comment]
this is blocked until we decide on a name for the service 

@CharliePatterson @Darren-CheckPlanningData @alextea

**Comment by CharliePatterson** [comment]
@GeorgeGoodall - my two cents below on the URL paths for these pages, though I think this doesn't quite match up with how things have been built?

The recommendation from Gwen on the content design side of things is to go with submit.planning.data.gov.uk.


| URL path                                                                                                           | Page                 | Description                                                                                                    |
|--------------------------------------------------------------------------------------------------------------------|----------------------|----------------------------------------------------------------------------------------------------------------|
| https://submit.planning.data.gov.uk/                                                                               | Root/Home            | Landing page that sets out the specifics of the service and what the user should expect                        |
| https://submit.planning.data.gov.uk/organisations                                                                  | Organisation finder  | Where the user can find their organisation                                                                     |
| https://submit.planning.data.gov.uk/organisations/local-authority:LBH/                                             | Dashboard            | The view of the datasets they need to provide and their progress towards this, as well as surfacing any issues |
| https://submit.planning.data.gov.uk/organisations/local-authority:LBH/tree-preservation-zone/                      | Dataset details page | The view of the data the LPA has provided                                                                      |
| https://submit.planning.data.gov.uk/organisations/local-authority:LBH/tree-preservation-zone/get-started           | Providing data flow  | A step-by-step guide of how to provide data                                                                    |
| https://submit.planning.data.gov.uk/organisations/local-authority:LBH/tree-preservation-zone/issue-task            | Issue tasklist       | A list of the issues within a dataset                                                                          |
| https://submit.planning.data.gov.uk/organisations/local-authority:LBH/tree-preservation-zone/issue-task/issue-type | Issues details page  | Details of the specific issue                                                                                  |
| https://check.planning.data.gov.uk/organisations/local-authority:LBH/tree-preservation-zone/error                  | Error details page   | Details of the error around accessing the endpoint                                                             |

**Comment by GeorgeGoodall** [comment]
need to wait on chris c / ernest to come back on the 27th to updated the domain and setup redirects

will update the frontend code for now but not merge it until infa is ready

**Comment by GeorgeGoodall** [comment]
having a meeting with Ernest and Chris today for which the agenda is:
- inform them of all the redirects
- inform them of the new domain
- coordinate going live with the new domain and deploying the new paths

Additional unrelated work:
- making use of the performance api / db

**Comment by GeorgeGoodall** [comment]
## Update
this has now been tested on development and we will orchestrating deployment to prod soon

**Comment by GeorgeGoodall** [comment]
Have just made another PR to address incorrect check header href 
https://github.com/digital-land/submit/pull/369
